amun/overlays/ocp4-stage/amun-api/kustomization.yaml

```diff
@@ -10,6 +10,8 @@ patchesStrategicMerge:
   - imagestreamtag.yaml
 generators:
   - ./secret-generator.yaml
+  - openshift-templates/inspection-workflow.yaml
+  - openshift-templates/inspection-with-cpu-workflow.yaml
```

Comment:
This should go under the patchStrategyMerge field as this would update the template defined in the base.